apps/docs/src/app/core/component-docs/input-group/examples/input-group-text-example.component.html

```diff
@@ -1,3 +1,5 @@
+<div fd-form-header>Form Header</div>
+<br>
```

Comment:
Why there is a `<br>`? Styles do not specify some spacing between header and the form elements?

Reply:
nope, no spacing is specified for the Header.